### PR TITLE
feat(entity): add bulk insert of id mappings

### LINKED ISSUE #14: Add bulk insert of id mappings

### PR DESCRIPTION
**Pull request by CptSchnitz**
closes #14

### CODE REVIEW

File: src/entity/models/entityManager.ts [hunk 1]
```diff
@@ -34,6 +34,26 @@ export class EntityManager {
...
+    });
+
+    if (dbEntity) {
+      const message = `an entity with the following ids: ${JSON.stringify(dbEntity)} already exists`;
+      throw new IdAlreadyExistsError(message);
+    }
+
+    await this.repository.insert(newEntities);
+  }
+
```
Comment (syncush) [inline src/entity/models/entityManager.ts:56]: does it insert in a single transaction or X transactions?

File: tests/integration/entity/entity.spec.ts [hunk 2]
```diff
@@ -83,6 +83,71 @@ describe('entity', function () {
     });
   });
 
+  describe('POST /entities', function () {
```
Comment (syncush) [inline tests/integration/entity/entity.spec.ts:86]: POST /entities/bulk ?

**Review by syncush — CHANGES_REQUESTED** [review]

File: src/entity/models/entityManager.ts (hunk 1 above)
Comment (CptSchnitz) [reply src/entity/models/entityManager.ts:56]: it inserts in a single transaction. here's an example from the query logger:
```
query: INSERT INTO "entity"("external_id", "osm_id") VALUES ($1, $2), ($3, $4) -- PARAMETERS: ["3fa8534-5717-4562-b3fc-2c963f26a1a6",1233,"3fa85f3-5717-4562-223fc-2c963f66afa6",1430]
```

File: tests/integration/entity/entity.spec.ts (hunk 2 above)
Comment (CptSchnitz) [reply tests/integration/entity/entity.spec.ts]: fixed

**Review by syncush — APPROVED** [review]

**Pull request merged**